Task.js

```diff
@@ -171,25 +169,31 @@ class Task extends Unit {
         break;
 
       case ContentTypes.ASSESMENT:
-      	  page_data.isAssesment = true;
-      if(shell.test("-f", this.sourcePath())){
-        page_data.questions = require(this.sourcePath());
-        page_data.questions_str = JSON.stringify(page_data.questions);
-	page_data.isJsonVersion = true;
-      }
-      else {
-        const jsonpath = this.sourcePath();
-	const jspath = path.resolve(path.dirname(jsonpath), `${path.basename(jsonpath, "json")}js`);
-	
-	if(shell.test("-f", jspath)){
-          page_data.quiz_src = path.basename(jspath);
-	  page_data.isJsVersion = true;
-	}
-	else {
-	  console.log(`${jspath} is missing`);
-	  process.exit(-1);
-	}
-      }
+        page_data.isAssesment = true;
+        if (shell.test("-f", this.sourcePath())) {
```

Comment:
I know it is not your code but it surprises me why we are using the shell functions when we already have the fs module loaded. This may be a future enhancement work. Please create an issue for this on the repo.